app/_hub/kong-inc/saml/_index.md

```diff
@@ -691,6 +694,33 @@ curl --request POST \
 3. If user credentials are valid, the browser will be redirected to https://httpbin.org/anything
 4. If the user credentials are invalid, a `401` unauthorized HTTP status code is returned
 
+## Troubleshooting
+
+### Certificate is valid, but isn't being accepted
+
+**Error:**
```

Comment:
I like the "Error"/"Solution" labels and sections, but I think reading "Error:" in this context was a little confusing. I thought it might precede an error message, but the error message was after that. 

I know it's a synonym, but maybe something like "Problem" would work? Just because I feel like "error" is more likely to be used in an error message. Or maybe you could add `idp_certificate` to the section header and have "Error message:" with the error message right after that (and remove the sentence before the error message) and then have "Solution:" like you have it now?

Reply:
I like the idea of "Problem" or "Issue" instead of error. "Problem" is also more consistent, we've done it here before: https://docs.konghq.com/konnect/org-management/troubleshoot/